package.json

```diff
@@ -47,7 +47,7 @@
     "@babel/register": "^7.13.0",
     "@loaders.gl/csv": "^3.0.0-alpha",
     "@loaders.gl/polyfills": "^3.0.0-alpha",
-    "@luma.gl/test-utils": "^8.4.1",
+    "@luma.gl/test-utils": "^8.5.0-alpha.1",
```

Comment:
Here we are opting for specific version. Any reason to be consistent?

Reply:
It doesn't matter. The master branch dependencies are controlled by yarn.lock. And when you install from npm, both resolve to the same result.